pkg/satellite/satellitedb/projectmembers_test.go

```diff
@@ -186,45 +186,54 @@ func TestProjectMembersRepository(t *testing.T) {
 	})
 }
 
-func prepareUsersAndProjects(ctx context.Context, users stlt.Users, projects stlt.Projects) ([]stlt.User, []stlt.Project) {
-	user1 := &stlt.User{
+func prepareUsersAndProjects(ctx context.Context, t *testing.T, users stlt.Users, projects stlt.Projects) ([]stlt.User, []stlt.Project) {
+	usersList := []*stlt.User{{
 		Email:        "email1@ukr.net",
 		PasswordHash: []byte("some_readable_hash"),
 		LastName:     "LastName",
 		FirstName:    "FirstName",
-	}
-	user2 := &stlt.User{
+	}, {
 		Email:        "email2@ukr.net",
 		PasswordHash: []byte("some_readable_hash"),
 		LastName:     "LastName",
 		FirstName:    "FirstName",
-	}
-	user3 := &stlt.User{
+	}, {
 		Email:        "email3@ukr.net",
 		PasswordHash: []byte("some_readable_hash"),
 		LastName:     "LastName",
 		FirstName:    "FirstName",
+	},
 	}
 
-	user1, _ = users.Insert(ctx, user1)
-	user2, _ = users.Insert(ctx, user2)
-	user3, _ = users.Insert(ctx, user3)
-
-	project1 := &stlt.Project{
-		Name:          "projName1",
-		TermsAccepted: 1,
-		Description:   "Test project 1",
-		OwnerID:       &user1.ID,
+	var err error
+	for i, user := range usersList {
+		usersList[i], err = users.Insert(ctx, user)
+		if err != nil {
+			t.Fatal(err)
+		}
 	}
-	project2 := &stlt.Project{
-		Name:          "projName2",
-		TermsAccepted: 1,
-		Description:   "Test project 1",
-		OwnerID:       &user2.ID,
+
+	projectList := []*stlt.Project{
+		{
+			Name:          "projName1",
+			TermsAccepted: 1,
+			Description:   "Test project 1",
+			OwnerID:       &usersList[0].ID,
+		},
+		{
+			Name:          "projName2",
+			TermsAccepted: 1,
+			Description:   "Test project 1",
+			OwnerID:       &usersList[1].ID,
+		},
 	}
 
-	project1, _ = projects.Insert(ctx, project1)
-	project2, _ = projects.Insert(ctx, project2)
+	for i, project := range projectList {
+		projectList[i], err = projects.Insert(ctx, project)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
 
-	return []stlt.User{*user1, *user2, *user3}, []stlt.Project{*project1, *project2}
+	return []stlt.User{*usersList[0], *usersList[1], *usersList[2]}, []stlt.Project{*projectList[0], *projectList[1]}
```

Comment:
`return usersList, projectList`

Reply:
omg, i should take a break :D 
i'm too inattentive